wlts/collections/collection_manager.py

```diff
@@ -103,24 +103,30 @@ def collections(self):
         return self._collections.values()
 
     def load_all(self):
-        """Creates all collection based on json of image and feature collection."""
-        json_string_feature = pkg_resources.resource_string('wlts', '/json_configs/feature_collection.json').decode(
-            'utf-8')
-
-        json_string_image = pkg_resources.resource_string('wlts', '/json_configs/image_collection.json').decode('utf-8')
-
-        config_feature = json_loads(json_string_feature)
-        config_image = json_loads(json_string_image)
-
-        if "feature_collection" in config_feature:
-            feature_collection = config_feature["feature_collection"]
-            for ft_collection in feature_collection:
-                self.insert("feature_collection", ft_collection)
-
-        if "image_collection" in config_image:
-            image_collection = config_image["image_collection"]
-            for img_collection in image_collection:
-                self.insert("image_collection", img_collection)
+        """Creates all collection based on json of feature and feature collection."""
+        import json
+        import os
+        from os import path
```

Comment:
You may omit this import and use directly `os.path`  instead (lines 114 and 123, etc.)